scalastyle-config.xml

```diff
@@ -35,7 +35,7 @@
   </check>
   <check enabled="true" class="org.scalastyle.scalariform.EmptyClassChecker" level="error"/>
   <check level="error" class="org.scalastyle.scalariform.NoWhitespaceBeforeLeftBracketChecker" enabled="true"/>
-  <check level="error" class="org.scalastyle.scalariform.NoWhitespaceAfterLeftBracketChecker" enabled="true"/>
```

Comment:
What is the reason for this change?

Reply:
Classes with a long list of type parameters which needed some linebreaks. Should not be neccessary anymore, so I reverted the change